### PR TITLE
CMR-5920: validate granule data format with KMS values

### PR DESCRIPTION
**Pull request by mschmele**

### CODE REVIEW

File: common-app-lib/src/cmr/common_app/services/kms_fetcher.clj [hunk 1]
```diff
@@ -74,6 +75,7 @@
   "Calls GCMD KMS endpoints to retrieve the keywords. Response is a map structured in the same way
   as used in the KMS cache."
   [context]
+  (def keyword-scheme :granule-data-format)
```
Comment (ygliuvt) [inline common-app-lib/src/cmr/common_app/services/kms_fetcher.clj]: Remove this line. Not sure why it is here.

File: ingest-app/src/cmr/ingest/services/messages.clj [hunk 2]
```diff
@@ -49,6 +49,15 @@
    (format "Data center short name [%s] was not a valid keyword."
           (:ShortName data-center)))
 
+(defn data-format-not-matches-kms-keywords
+  "Error message when FileDistributionInformation -> Format is
+  not in the KMS."
+  [archive-and-distribution-information]
+  (format "Format [%s] within ArchiveAndDistributionInformation [%s] was not a valid keyword."
```
Comment (ygliuvt) [inline ingest-app/src/cmr/ingest/services/messages.clj]: I don't think it is a good error message. The ArchiveAndDistributionInformation might be too much for an error message.

Comment (ygliuvt) [reply ingest-app/src/cmr/ingest/services/messages.clj]: Can we add a test for validating granule data format with KMS values? Then we can see what this error message really looks like.

File: common-app-lib/src/cmr/common_app/services/kms_fetcher.clj (hunk 1 above)
Comment (mschmele) [reply common-app-lib/src/cmr/common_app/services/kms_fetcher.clj]: It was for debugging purposes and I just missed it. Fixed

File: ingest-app/src/cmr/ingest/services/messages.clj (hunk 2 above)
Comment (mschmele) [reply ingest-app/src/cmr/ingest/services/messages.clj]: Fixed and added tests

**Pull request merged**